### DISCUSSION TITLE
Org-wide Required workflows - feature requests

### DISCUSSION THREAD

**Discussion by chejn** [community/community#45484] [Actions]
### Select Topic Area

Product Feedback

### Body

Hey all,
It would be nice if the following were added as features to the newly introduced org-wide requirements,

- Org-wide workflows, but not required as a status check. There are alot of instances where users would like to apply workflows across all repositories but do not want to enforce them as a status check.
- Allow workflows from other organizations, currently this is limited to any workflow in the same org.

**Comment by reiniertimmer** [comment]
Regarding your first point, see discussion here: https://github.com/orgs/community/discussions/43890

It would be great to have this feature, it would make organization workflows much more powerful

**Comment by avivek** [comment]
Another item that would be useful for sure is to not limit the triggers just to pull request.
We have tried to use on push trigger on the default branch in the required workflow triggers, but for some reason these events do not trigger the workflow.

Also this makes the action cache totally unusable for the required workflow as the default branch never gets to have these actions run on them.

https://github.com/actions/cache/issues/79#issuecomment-1474819386

**Comment by FritzerMcQ** [comment]
Would also be great if there would be the possibility to bypass them like usual branch-protection rules.
